.devcontainer/devcontainer.json

```diff
@@ -17,16 +23,13 @@
 			"version": "latest",
 			"dockerDashComposeVersion": "v1"
 		}
-	}
-
-	// Features to add to the dev container. More info: https://containers.dev/features.
-	// "features": {},
+	},*/
 
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "python --version",
+	"postCreateCommand": "pip install -r requirements.txt"
```

Comment:
we're using Anaconda...do you need this at all?

Reply:
`make serve-local` didn't work because of requirements not downloaded?